### ISSUE TITLE
Got Error: "Cannot GET /"

### ISSUE THREAD

**Issue by inix** [abraham/sprinkle#1]
I have cloned code and install require dependencies,I got following error when I navigated to Safari browser:
![wx20180119-155041](https://user-images.githubusercontent.com/2264291/35140055-c5a272a4-fd30-11e7-9105-b997c4f8396f.png)
![wx20180119-155101](https://user-images.githubusercontent.com/2264291/35140060-ca93c3c6-fd30-11e7-8d62-959e43bb586e.png)

**Comment by inix** [comment]
`"npm build" `command throw following errors:
```shell
$ gulp build
[23:32:52] Using gulpfile D:\src\wechat\sprinkle\gulpfile.js
[23:32:52] Starting 'copy'...
[23:32:52] Starting 'polyfills'...
>> [gulp-polyfiller] File "polyfills.js" created.
[23:32:52] Finished 'polyfills' after 98 ms
[23:32:52] Starting 'js'...

events.js:183
      throw er; // Unhandled 'error' event
      ^
Error: Couldn't find preset "es2015" relative to directory "D:\\src\\wechat\\sprinkle"
    at D:\src\wechat\sprinkle\node_modules\babel-core\lib\transformation\file\options\option-manager.js:293:19
    at Array.map (<anonymous>)
    at OptionManager.resolvePresets (D:\src\wechat\sprinkle\node_modules\babel-core\lib\transformation\file\options\option-manager.js:275:20)
    at OptionManager.mergePresets (D:\src\wechat\sprinkle\node_modules\babel-core\lib\transformation\file\options\option-manager.js:264:10)
    at OptionManager.mergeOptions (D:\src\wechat\sprinkle\node_modules\babel-core\lib\transformation\file\options\option-manager.js:249:14)
    at OptionManager.init (D:\src\wechat\sprinkle\node_modules\babel-core\lib\transformation\file\options\option-manager.js:368:12)
    at File.initOptions (D:\src\wechat\sprinkle\node_modules\babel-core\lib\transformation\file\index.js:212:65)
    at new File (D:\src\wechat\sprinkle\node_modules\babel-core\lib\transformation\file\index.js:135:24)
    at Pipeline.transform (D:\src\wechat\sprinkle\node_modules\babel-core\lib\transformation\pipeline.js:46:16)
    at DestroyableTransform._transform (D:\src\wechat\sprinkle\node_modules\gulp-babel\index.js:37:20)
```

I solved by executing: `npm install babel-cli babel-preset-es2015`

**Comment by abraham** [comment]
Closed in #2
